src/Domain/EmbeddingMaps/AffineMap.hpp

```diff
@@ -0,0 +1,97 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines the class AffineMap.
+
+#pragma once
+
+#include <memory>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/EmbeddingMaps/EmbeddingMap.hpp"
+#include "Parallel/CharmPupable.hpp"
+
+namespace EmbeddingMaps {
+
+/// Linear map from \f$\xi \in [A, B]\rightarrow x \in [a, b]\f$.
+/// The formula for the mapping is...
+/*! \f[
```

Comment:
I really don't like changing Doxygen styles in the middle of the comment. This seems like insanity to me... I think we even agreed either is fine, but not mixed, so this should be moved to `/*!` completely. Also, should have the ` *` beginning on the lines